api-docs/api-operations/methods/patch-attach-configuration-group-to-ha-instance-version-accountid-ha-haid.rst

```diff
@@ -0,0 +1,312 @@
+
+.. _patch-attach-configuration-group-to-ha-instance-version-accountid-ha-haid.rst:
+
+Attach Configuration Group to HA Instance 
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. code::
+
+    PATCH /{version}/{accountId}/ha/{haId}
+
+Attaches a specified configuration group to the HA Instance (See :ref:`post-create-configuration-version-accountid-configurations`).
+
+.. note::
+   If the configuration group has non-dynamic configuration parameters, the HA instance will be put in a ``RESTART_REQUIRED`` state. To enable the parameters on all the nodes (source and replicas of the HA group), restart the HA instance (see :ref:`post-restart-ha-instance-version-accountid-ha-haid-action`.).
+
```

Comment:
To enable the parameters on all the nodes (source and replicas of the HA group), :ref:`restart the HA instance <post-restart-ha-instance-version-accountid-ha-haid-action>`.